plugins/modules/credential.py

```diff
@@ -201,6 +221,20 @@ def main() -> None:
         else (controller.get_item_name(credential) if credential else name)
     )
 
+    if copy_from:
+        if not credential:
```

Comment:
WDYT if we encapsulate the copy_from logic in a function?

Reply:
I think that's a good idea. I tried to follow the existing pattern, let me know what you think.